README.md

```diff
@@ -1,4 +1,4 @@
-<p align="center"><img alt="Differencify" src="http://i.imgur.com/D0Eapjx.png" width="150">
+<p align="center"><img alt="Differencify" src="http://i.imgur.com/0zwMggK.png" width="150">
```

Comment:
Yeah, I see your point, but that applies when those needs are affecting the functionality of the project. The readme is more about documentation and if some pictures are missing disaster wouldn't happen. This pattern of putting image files externally is very popular in lots of projects. but I will change this. will put them in images folder